docs/.vale/thothtech/CodeblockFences.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Error: thothtech.CodeblockFences
+#
+# Ensures all codeblock language tags use the full name, not aliases.
+#
+# For a list of all options, see https://errata-ai.gitbook.io/vale/getting-started/styles
+extends: existence
+message: 'Syntax highlighting hint "%s" must be one of: yaml, ruby, plaintext, markdown, javascript, shell, golang, python, dockerfile, or typescript.'
+link: TBA
+level: error
+scope: raw
+raw:
+  - '\`\`\`(yml|rb|text|md|bash|sh\n|js\n|go\n|py\n|docker\n|ts)'
```

Comment:
Future proofing: will mermaid be considered an error?
If we are to promote the use of it in our docs, we don't want it to produce errors

Reply:
Thanks, @talzid. This rule only highlights issues with the listed short alias. `mermaid` does not have a short version, or if there is, this rule will not pick it up.